src/cli.zig

```diff
@@ -1,16 +1,18 @@
 const std = @import("std");
 const assert = std.debug.assert;
 const fmt = std.fmt;
+const math = std.math;
 const mem = std.mem;
 const meta = std.meta;
 const net = std.net;
 const os = std.os;
 
 const config = @import("config.zig");
+const tigerbeetle = @import("tigerbeetle.zig");
 const vsr = @import("vsr.zig");
 const IO = @import("io.zig").IO;
 
-// TODO Document --memory
+// TODO Document --cache-accounts, --cache-transfers, --cache-transfers-posted
```

Comment:
A good question is how we actually document this. I have a gut feeling that, over time, we'll end up with infinitely long help. Perhaps we should go ahead and do the per-subcommand help a-la zig itself?